src/schema/rules/checks/events.yaml

```
@@ -43,3 +43,15 @@ SortedOnsets:
   checks:
     # n/a values will likely cause false alarms if encountered. Consider alternatives.
     - allequal(sorted(columns.onset, "numeric"), columns.onset)
+
+NonNegativeOnsets:
+  issue:
+    code: EVENT_ONSET_NONNEGATIVE
+    message: |
+      The onsets are typically positive: negative ones are allowed but rarely encountered.
+    level: warning
+  selectors:
+    - suffix == "events"
+    - extension == ".tsv"
+  checks:
+    - min(columns.onset) >= 0
```

If we want to consider making it an error in a future release (e.g., 2.0), I would write it in a way that it can be promoted for testing and then switched in a future release. For example:

```suggestion
    code: IMPLAUSIBLE_EVENT_ONSET
    message: |
      Event onsets were found more than 60s before the start of recording, indicated by negative values.
      If this is not a mistake, remove or consider an alternative method of representing these events.
    level: warning
  selectors:
    - suffix == "events"
    - extension == ".tsv"
  checks:
    - min(columns.onset) >= -60
```

---

Fine with me... I guess we better also then tune up the test "name" (NonNegativeOnsets) to match the error code?  BTW -- why they are not harmonized: do we ever refer to the names somewhere?